packages/@aws-cdk/aws-iot-actions/lib/lambda-function-action.ts

```diff
@@ -12,7 +13,7 @@ export class LambdaFunctionAction implements iot.IAction {
   constructor(private readonly func: lambda.IFunction) {}
 
   bind(topicRule: iot.ITopicRule): iot.ActionConfig {
-    this.func.addPermission('invokedByAwsIotRule', {
+    this.func.addPermission(`${Names.nodeUniqueId(topicRule.node)}:Permission`, {
```

Comment:
I would make the `"Permission"` part more module-specific. Something like:

```suggestion
    this.func.addPermission(`${Names.nodeUniqueId(topicRule.node)}:IotLambdaFunctionAction`, {
```